test/blackbox-tests/test-cases/install-mld-pages/files-in-documentation-stanza.t/doc/examples/example1/index.mld

```diff
@@ -0,0 +1 @@
+{0 Example 1}
```

Comment:
Nitpick (along with some of the other files): Maybe worth adding a newline here to prevent the next person opening this file saving it with a newline.

Reply:
I though I had done those files, thanks for repeating! Fixed in eeac346bc